mlos_bench/mlos_bench/optimizers/mlos_core_optimizer.py

```diff
@@ -35,15 +35,15 @@ def __init__(self, tunables: TunableGroups, service: Optional[Service], config:
         space = tunable_groups_to_configspace(tunables)
         _LOG.debug("ConfigSpace: %s", space)
 
-        opt_type = getattr(OptimizerType, self._config.pop('optimizer_type', 'SKOPT'))
+        opt_type = getattr(OptimizerType, self._config.pop('optimizer_type', DEFAULT_OPTIMIZER_TYPE.name))
```

Comment:
Let `mlos_core` define what the default optimizer to use is.  Reduces our maintenance costs of having to update it in multiple places.